app/src/main/java/in/avimarine/boatangels/fragments/MyBoatFragment.java

```diff
@@ -81,18 +71,12 @@ public MyBoatFragment() {
   /**
    * Use this factory method to create a new instance of
    * this fragment using the provided parameters.
-   *
-   * @param param1 Parameter 1.
-   * @param param2 Parameter 2.
+
    * @return A new instance of fragment MyBoatFragment.
    */
   // TODO: Rename and change types and number of parameters
```

Comment:
TODELETE (all comments related to "Param"s)

Reply:
Done